frontend/src/features/gallery/ImageMetaDataViewer/ImageMetadataViewer.tsx

```diff
@@ -217,6 +218,13 @@ const ImageMetadataViewer = memo(
                   onClick={() => dispatch(setWidth(seamless))}
                 />
               )}
+              {hires_fix && (
+                <MetadataItem
+                  label="High Resolution Optimization"
+                  value={hires_fix}
+                  onClick={() => dispatch(setWidth(hires_fix))}
```

Comment:
This should be `setHiresFix()`

Reply:
@psychedelicious thanks for the review!

That makes sense to me, but had mostly been using the “seamless” implement as reference. 

Is the seamless code (above it) also supposed to be setSeamless? Or how do these two differ

Thx - still figuring a few things out in the implementation :)

Reply:
You are correct, it should be setSeamless. These are the buttons to recall individual parameters

Reply:
The calls are now broken in a different way hehe gotta provide the function with arguments

Reply:
Getting there. ;) 

Updated to include (what I hope) are the correct arguments